app/controllers/qualifications/users/sign_in_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Qualifications
+  module Users
+    class SignInController < QualificationsInterfaceController
+      skip_before_action :authenticate_user!
+      skip_before_action :handle_expired_token!
```

Comment:
Suggestion: we could redirect them straight to the quals show page if there's a `current_user`.

Reply:
Yep... agreed. I might do that in a separate PR for Jan to limit the changes we're doing this late in the day. The current start page doesn't do that so we're not making anything worse by not doing it now.